cmd/scaffold.go

```diff
@@ -573,11 +574,22 @@ func (fnb *FlowNodeBuilder) initLogger() {
 	// configure logger with standard level, node ID and UTC timestamp
 	zerolog.TimeFieldFormat = time.RFC3339Nano
 	zerolog.TimestampFunc = func() time.Time { return time.Now().UTC() }
+
+	// Drop all log events that exceed this rate limit
+	throttledSampler := &zerolog.BurstSampler{
+		Burst:  fnb.BaseConfig.debugLogLimit,
+		Period: time.Second,
+	}
+
```

Comment:
Please also add a similar `throttledSampler` dedicated for the [score option](https://github.com/onflow/flow-go/blob/57957a344efde12efffabbcbdb2d7b9e7ffd6edd/network/p2p/scoring/score_option.go#L96). It currently is triggering the logging per received message, but the content of log is mostly the same under the normal and honest operation. I think a burst of 100 within one second is sufficient for the score option. In this way we also safeguard the global logger against the score option spamming it. This is the only place of my knowledge that we experienced spamming at the networking layer.

Reply:
b21e90d